helper.py

```diff
@@ -1,12 +1,338 @@
 import random
 import string
+import clipboard
+import cli_ui
 
+def clear_screen():
+    print('\033[H\033[J')
 
-def password_generator(length):
-    sample_space = string.ascii_letters + string.digits + string.punctuation
-    password = ''.join((random.choice(sample_space) for i in range(length)))
-    return password
+def password_generator_main():
+    condition = True
 
+    while condition:
+        type1 = cli_ui.ask_yes_no("Do you want Upper Case Characters in your password?")
+        type2 = cli_ui.ask_yes_no("Do you want Lower Case Characters in your password?")
+        type3 = cli_ui.ask_yes_no("Do you want Special Characters in your password?")
+        type4 = cli_ui.ask_yes_no("Do you want Numbers in your password?")
```

Comment:
Add these prompts should be asked in main and their results should be passed as arguments to the password_generator function